project-base/src/Controller/Front/CartController.php

```diff
@@ -79,7 +86,8 @@ public function __construct(
         OrderPreviewFactory $orderPreviewFactory,
         ErrorExtractor $errorExtractor,
         ListedProductViewFacadeInterface $listedProductViewFacade,
-        RequestStack $requestStack
+        RequestStack $requestStack,
+        ?ModuleFacade $moduleFacade = null
```

Comment:
I am sorry, I have urgent need to confirm something...
the question mark at the beginning of line is on keyboard very close to uppercase M.
Perhaps it is a typo?